docs/guides/job-limits.mdx

```diff
@@ -8,41 +8,21 @@ description: Job limits imposed by the job validation service for jobs sent to I
 
 <Admonition type="note">
-Certain primitive options increase the circuit size.  The described limits are checked _after_ the expected increase in circuit size. In particular, these options increase circuit size:
-  - Dynamical decoupling and gate-folding ZNE introduce additional gates that are included in the instructions for the [Maximum number of low-level instructions per qubit](#per-qubit) limit.
-  - Gate-folding ZNE introduces additional two-qubit gates relevant to the [Maximum number of two-qubit gates per job](#2-qubit-limit) limit. The number of two-qubit gates is multiplied by the sum of noise factors requested in gate-folding ZNE.
+Certain primitive options increase the circuit size. The described limits are checked *after* the expected increase in circuit size. In particular, these options increase circuit size:
+
+- Dynamical decoupling and twirling introduce additional single-qubit gates.
+- Gate-folding ZNE introduces additional two-qubit gates. The number of two-qubit gates is multiplied by the sum of noise factors requested in gate-folding ZNE.
 </Admonition>
 
 <span id="max-shots"></span>
 ## Maximum executions
 
-At most, **5 million** executions are allowed.  The number of executions is the number of circuits times the number of shots, where the circuits are those generated after PUB elements are broadcasted.
+At most, **10 million** executions are allowed.  The number of executions is the number of circuits times the number of shots, where the circuits are those generated after PUB elements are broadcasted.
 
 For example, if you have a PUB with one circuit, observables with shape (1, 6), and parameters with shape (4, 1), this would render $4 \times 6 = 24$ circuits (or fewer, if some observables commute). If you requested 2,000 shots, then the total number of executions is $24 \times 2,000 = 48,000$.
 
-<span id="per-qubit"></span>
-## Maximum number of low-level instructions per qubit
-
-The service permits up to **26.8 million control-system instructions per qubit**. This ensures that the user circuits fit within the control system's instruction memory.  The following table describes how the system translates instruction set architecture (ISA) circuit instructions to control system instructions when calculating this limit.
-
-| Instruction | Count |
-|-------------|-------|
-| `rz`          | 1     |
-| `delay`       | 1     |
-| `sx`          | 2     |
-| `x`           | 2     |
-| `cx`          | 5     |
-| `cz`          | 5     |
-| `ecr`         | 5     |
-| `measure`     | 10    |
-| `reset`       | 17    |
-| `init`        | 50    |
-
-<Admonition type="note">
-  This table captures the heuristic used in validation and does not reflect the exact number of instructions used to implement an operation.
-</Admonition>
-
 <span id="2-qubit-limit"></span>
-## Maximum number of two-qubit gates per job
+## Maximum number of two-qubit gates per circuit
```

Comment:
I think we can move the Note to this section, since it only talks about gate limits. Also we might as well include the limits for single-qubit gates as well (in addition to 2q gates).